app/assets/stylesheets/application.scss

```diff
@@ -136,42 +123,46 @@ h5 {
 .content {
   background-color: $white;
   box-shadow: 5px 5px 5px $shadow-color;
-  margin: 0 0 30px;
+  margin: 0;
   padding: 0;
 }
 
 .user-index {
   list-style: none;
   text-align: left;
-  margin: 0;
   padding: 20px;
 }
 
+.profile-name {
+  margin-top: 15px;
+  font-weight: bold;
+  font-size: 1em;
+}
+
 .user-name {
+  margin-top: 0px;
```

Comment:
Properties should be ordered font-size, font-weight, margin-top<br>`0px` should be written without units as `0`